package l0;

import android.hardware.camera2.params.SessionConfiguration;
import java.util.Iterator;
import java.util.List;

/* JADX INFO: loaded from: classes.dex */
final class a implements d {

    /* JADX INFO: renamed from: a, reason: collision with root package name */
    private final List<d> f113939a;

    a(List<d> list) {
        this.f113939a = list;
    }

    @Override // l0.d
    public d.a a(SessionConfiguration sessionConfiguration) {
        Iterator<d> it = this.f113939a.iterator();
        while (it.hasNext()) {
            d.a aVarA = it.next().a(sessionConfiguration);
            if (aVarA.a() != 0) {
                return aVarA;
            }
        }
        return new d.a(0, 0, 0L);
    }
}
